src/flowMC/sampler/Sampler.py

```diff
@@ -7,36 +7,18 @@
 import optax
 from flowMC.sampler.Proposal_Base import ProposalBase
 from flowMC.nfmodel.base import NFModel
+from flowMC.utils import initialize_summary_dict
 from tqdm import tqdm
 import equinox as eqx
 import numpy as np
-
+import matplotlib.pyplot as plt
+from flowMC.utils.hyperparameters import flowmc_default_hyperparameters
 
 class Sampler:
     """
     Sampler class that host configuration parameters, NF model, and local sampler
 
-    Args:
```

Comment:
All hyper-parameters doc strings should remain where it is for the doc to render correctly. Also instead of referring to more files, I rather have the parameters where they are called